src/axom/quest/examples/quest_proe_bbox.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*! \file quest_proe_bbox.cpp
+ *  \brief This example code demonstrates how to read a subset of a ProE (Creo)
+ *  text-format tetrahedron mesh using a bounding box.  It reads a file
+ *  specified with the -f argument and writes the mesh contained therein
+ *  into a file specified with the -o argument.
+ */
+
+// Axom includes
+#include "axom/config.hpp"
+#include "axom/core.hpp"
+#include "axom/slic.hpp"
+
+#include "axom/CLI11.hpp"
+#include "axom/fmt.hpp"
+
+// _read_proe_include2_start
+#include "axom/mint/mesh/Mesh.hpp"
+#include "axom/mint/mesh/UnstructuredMesh.hpp"
+#include "axom/mint/utils/vtk_utils.hpp"  // for write_vtk
+// _read_proe_include2_end
+
+// _read_proe_include1_start
+#include "axom/quest/readers/ProEReader.hpp"
+// _read_proe_include1_end
+
+// _read_proe_typealiases_start
+namespace mint = axom::mint;
+namespace primal = axom::primal;
+namespace slic = axom::slic;
+
+using IndexType = axom::IndexType;
+using UMesh = mint::UnstructuredMesh<mint::SINGLE_SHAPE>;
+// _read_proe_typealiases_end
+
+//------------------------------------------------------------------------------
+void initialize_logger()
+{
+  // initialize logger
+  slic::initialize();
+  slic::setLoggingMsgLevel(slic::message::Info);
+
+  // setup the logstreams
+  std::string fmt = "";
+  slic::LogStream* logStream = nullptr;
+
+  fmt = "[<LEVEL>]: <MESSAGE>\n";
+  logStream = new slic::GenericOutputStream(&std::cout, fmt);
+
+  // register stream objects with the logger
+  slic::addStreamToAllMsgLevels(logStream);
+}
+
+//------------------------------------------------------------------------------
+void finalize_logger()
+{
+  slic::flushStreams();
+  slic::finalize();
+}
+
+struct Arguments
+{
+  std::string file_name;
+  std::string outfile_name;
+
+  void parse(int argc, char** argv, axom::CLI::App& app)
+  {
+    app
+      .add_option("-f,--file", this->file_name, "specifies the input mesh file")
+      ->check(axom::CLI::ExistingFile)
+      ->required();
```

Comment:
Suggestion: For consistency w/ other axom examples, please consider naming this parameter `-i,-input`

Reply:
I copied this from `quest_bvh_two_pass.cpp`, which I don't plan to change.

Reply:
(I did rename this file's options.)